packages/craco/README.md

```diff
@@ -313,11 +401,14 @@ const webpackConfig = createWebpackDevConfig(cracoConfig);
 module.exports = webpackConfig;
 ```
 
-### Popular integrations
 
+## Recipes and Plugins
+### Recipes
 - [vscode-jest](https://github.com/sharegate/craco/tree/master/recipes/use-a-jest-config-file)
```

Comment:
Not sure about this here.

The purpose of this integreation was to showcase how to exported the resulting Jest config to integrate with another tool.

It is very specific to the API section and should be moved back to API and I think labeled with something like "Integrations" or "Popular integrations"

Also, I don't really see any similarities between "Recipes" and "Plugins". I am not convinced they should be merged together.

Reply:
You're right. I didn't spend time on this section. I have now, though and it is more carefully outlined. See refactor.